addon/components/nrg-dropdown/component.js

```diff
@@ -290,6 +290,16 @@ export default Component.extend(Validation, EKMixin, EKFirstResponderOnFocusMixi
   },
 
   mouseDown(evt) {
+    const dropdownElement = evt.target.closest('.dropdown');
+
+    if (dropdownElement) {
+        const rect = dropdownElement.getBoundingClientRect();
+        if (evt.clientX > rect.left + rect.width || evt.clientY > rect.top + rect.height) {
+            return;
+        }
+    }
+
+
```

Comment:
I'm not sure I'm following the need for this. If the user clicks on something inside the dropdown element that isn't a dropdown item, nothing should happen. What currently happens when you click on the scrollbar? What path does the function take? Does the end of this function need to `evt.preventDefault()`?

Reply:
Well there's a card in jira, and when the user clicks on the scrollbar, it closes the dropdown. Still trying to figure out how to make the filter headings in the directory non-clickable, because they will close the modal as well.